install_build_dependencies.sh

```diff
@@ -47,7 +48,11 @@ if [ -f /etc/lsb-release ]; then
             libtool \
             autoconf \
             shellcheck \
-            python \
+            patchelf \
+            libenchant1c2a \
+            python3-pip \
```

Comment:
should not we use https://pip.pypa.io/en/stable/installation/#get-pip-py approach?

Reply:
Why do you need to download and compile cmake when OS cmake is suitable???

Reply:
patchelf libenchant1c2a is not python
pip is needed for installing using pip :)))